ingest/src/main/java/com/microsoft/azure/kusto/ingest/ColumnMapping.java

```diff
@@ -0,0 +1,30 @@
+package com.microsoft.azure.kusto.ingest;
+
+import java.io.Serializable;
+import java.util.Map;
+
+/**
+ * Data class that describes the Mapping setting for .ingest command
+ */
+public class ColumnMapping implements Serializable {
+    public String columnName;
+    public String columnType;
+    public Map<String, String> properties;
+
+    public ColumnMapping(String columnName, String cslDataType, Map properties) {
+        this.columnName = columnName;
+        this.columnType = cslDataType;
+        this.properties = properties;
+    }
+
+    public void addProperty(MappingConst propertyName, String value){
```

Comment:
shouldn't we have specific setters and getters for specific keys? it's not like ColumnMapping can hold any random key

Reply:
it's not. propertyName is of type MappingConst .. 
the .Net implementation doesnt even force the key to be of specific type ...

Reply:
I mean what you are doing is basically achieving the same result with less readability. we should improve the .Net version as well.